File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/repository/SedMottattRepository.java
package no.nav.melosys.eessi.repository;

import java.util.Collection;

import no.nav.melosys.eessi.models.SedMottatt;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * Erstattet av SedMottattHendelseRepository
 */
@Deprecated(forRemoval = true)
public interface SedMottattRepository extends JpaRepository<SedMottatt, Long> {
    Collection<SedMottatt> findAllByFerdigFalseAndFeiletFalse();

    Double countByFeiletIsTrue();
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/repository/SedMottattRepository.kt
package no.nav.melosys.eessi.repository

import no.nav.melosys.eessi.models.SedMottatt
import org.springframework.data.jpa.repository.JpaRepository

/**
 * Erstattet av SedMottattHendelseRepository
 */
@Deprecated("")
interface SedMottattRepository : JpaRepository<SedMottatt, Long> {
    fun findAllByFerdigFalseAndFeiletFalse(): MutableCollection<SedMottatt>

    fun countByFeiletIsTrue(): Double?
}
